### PR TITLE
[expo-sqlite] accept null for arguments in `executeSqlAsync`

### PR DESCRIPTION
**Pull request by spencerc99**
# Why

#18078 fixed this for the callback-based functions, but not for `executeSqlAsync`. Changing the parameters for the `args` of that function to match.

# How

just a type change. I made the change to my local expo code and it works.

# Checklist

I'm not sure how to update the documentation on docs.expo.dev, happy to do so if i get a pointer

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Comment by alanjhughes** [comment]
Hi, thanks for your contribution!
You'll also need to make the change [here](https://github.com/expo/expo/blob/48235c2fc7befd5f0e821e38be2e299d8302ebdf/packages/expo-sqlite/src/SQLite.ts#L267) and rebuild the js files
```bash
cd packages/expo-sqlite
yarn build
```
And also add a changelog? Once those things are done, I can merge.

**Comment by spencerc99** [comment]
> ```shell
> cd packages/expo-sqlite
> ```

thanks! updated. There was also a small `yarn.lock` change when I built FYI (last commit)

**Comment by spencerc99** [comment]
> Sorry for the delay, can you revert the yarn change and rebase?

done

**Comment by spencerc99** [comment]
> Sorry for the delay, can you revert the yarn change and rebase?

hey there anything else needed for this?

File: yarn.lock [hunk 1]
```diff
@@ -12076,7 +12076,7 @@ isurl@^1.0.0-alpha5:
     has-to-string-tag-x "^1.2.0"
     is-object "^1.0.1"
 
-jackspeak@^2.3.5:
+jackspeak@^2.0.3, jackspeak@^2.3.5:
```
Comment (Kudo) [inline yarn.lock:12069]:
```suggestion
jackspeak@^2.3.5:
```

File: packages/expo-sqlite/CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,7 @@
 ### 🐛 Bug fixes
 
 - Fix crash issues in `sqlite/next`. ([#25295](https://github.com/expo/expo/pull/25295) by [@kudo](https://github.com/kudo))
+- fix `executeSqlAsync` to accept null properly as arguments. ([#24761](https://github.com/expo/expo/pull/24761) by [@spencerc99](https://github.com/spencerc99))
```
Comment (Kudo) [inline packages/expo-sqlite/CHANGELOG.md:12]:
```suggestion
- Fix `executeSqlAsync` to accept null properly as arguments. ([#24761](https://github.com/expo/expo/pull/24761) by [@spencerc99](https://github.com/spencerc99))
```

**Review by Kudo — APPROVED** [review]
thanks much for the fix 🎉

**Pull request merged**